README/ReleaseNotes/v626/index.md

```diff
@@ -287,4 +299,4 @@ The class version of `RooAbsArg` was incremented from 7 to 8 in this release. In
 ## PyROOT
 
 - The `ROOT` Python module is now properly serializable so that it is automatically available in the Python environment if a function or ROOT object needs to be serialized. See issue [#6764](https://github.com/root-project/root/issues/6764) for a concrete usecase.
-- Improve overload resolution of functions that accept classes with long inheritance trees. Now prefer to call the function overload of the most derived class type (PR [#9092](https://github.com/root-project/root/pull/9092)).
\ No newline at end of file
+- Improve overload resolution of functions that accept classes with long inheritance trees. Now prefer to call the function overload of the most derived class type (PR [#9092](https://github.com/root-project/root/pull/9092)).
```

Comment:
Is this change by error?

Reply:
I tried to avoid this diff that shows no difference, but I didn't manage to! I think it might have something to do with the file ending that is differently encoded on Linux when saving the file.